### PR TITLE
Boycott Page - Uyghur

### PR DESCRIPTION
**Pull request by ms858**
possible change: add @pageBackground to Table color (?)

### CODE REVIEW

File: src/components/uyghur/boycott/index.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+
+// semantic-ui
+import {
+    Container, Icon, Label, Menu, Table 
+  } from 'semantic-ui-react';
+
+class UyghurBoycott  extends Component {
+  render() {
+    const color = site.variables.get(pageBackground);
```
Comment (amanmibra) [inline src/components/uyghur/boycott/index.jsx:10]: What color were you intending on using for the table? This throws an error as `site` and `pageBackground` are undeclared variables.

File: src/components/uyghur/boycott/index.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        </p>
+        <p class= "quote"> <i>  
+            “The Chinese government has facilitated the mass transfer of Uyghur and other ethnic minority citizens from the far west region of Xinjiang to factories across the country. Under conditions that strongly suggest forced labour, Uyghurs are working in factories that are in the supply chains of at least 83 well-known global brands in the technology, clothing and automotive sectors, including Apple, BMW, Gap, Huawei, Nike, Samsung, Sony and Volkswagen.”
+        </i>
+        </p>
+        <br></br>
+        <p> <b>
+            List of orgs to boycott:</b>
+        </p>
+        <Table celled color= {color} key ="#F9F9F9" inverted>
```
Comment (amanmibra) [inline src/components/uyghur/boycott/index.jsx:24]:
```suggestion
        <Table celled color="primary" key ="#F9F9F9" inverted>
```

I suggest just using the default primary or secondary colors that I already have set up with semantic. If there is a specific color that you think would look much better, I can add it to the semantic build. But I would still highly recommend just using those defaults.

**Comment by ms858** [comment]
Oh shoot, I thought I discarded all the color-related changes

**Comment by amanmibra** [comment]
@ms858 suggestion on branch naming conventions -> check out this read: https://gist.github.com/digitaljhelms/4287848

Not a huge deal, but generally just try to name all of branches lowercased, and add your username after (this is useful especially if you or someone else needs to work on a branch that is related to boycott). So maybe `boycott-uyghur-ms858` would have been great! But no worries, `Boycott` works for now! Good job on getting this done!

File: src/components/uyghur/boycott/index.jsx (hunk 2 above)
Comment (ms858) [reply src/components/uyghur/boycott/index.jsx:24]: yes, I was looking for what those are called! what is the cream-yellow bg color called?

Comment (amanmibra) [reply src/components/uyghur/boycott/index.jsx:24]: Our color palette can be found here! Sorry for not showing you that yet, but there is a folder called Website Layout which also has other things related to our site like our logos for instance

https://docs.google.com/document/d/1B86r3P9PZ7VHmQqorNP4MCg9G2srR1dl0ljghzOfqyk/edit?usp=sharing

**Comment by ms858** [comment]
I will edit

**Comment by amanmibra** [comment]
Good work @ms858! I will go ahead and merge this, but take note of my commit into your branch ^ I made small adjustments to tabbing and other things like our imports.

**Pull request merged**